plugins/wildfly_logs.yaml

```
@@ -0,0 +1,103 @@
+version: 0.0.1
+title: Wildfly
+description: Log parser for Wildfly
+parameters:
+  - name: standalone_file_path
+    type: "[]string"
+    default:  
+      - "/opt/wildfly/standalone/log/server.log"
+  - name: enable_domain_server
+    type: bool
+    default: false
```

I think both this `enable_domain_server` and `enable_domain_controller` should be default true.
```suggestion
    default: true
```